### ISSUE TITLE
Will you release Pretrained Model based on RGB and ToTensor() Inputs ?

### ISSUE THREAD

**Issue by Alexandmrwh** [megvii-model/ShuffleNet-Series#15]
Great works! The pretrained model you release is based on bgr input (0-255), will you release the rgb input (transformed by pytorch) ?

**Comment by zhjpqq** [comment]
Why don't use the ToTensor() in PyTorch? 

use ToBGRTensor()  and OpenCVReize() for much faster ? or for better accuracy ?

**Comment by nmaac** [comment]
Hi @Alexandmrwh , the models we trained are based on BGR input, you may need to adjust your input format if you use the pretrained models.

Hi @zhjpqq , it doesn't affect speed and accuracy, we use it just to keep the data format consistent with our inner framework's format.
